android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,11 @@
         <provider android:authorities="${applicationId}.fileprovider" android:exported="false" android:grantUriPermissions="true" android:name="androidx.core.content.FileProvider">
             <meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/file_paths" />
         </provider>
+        <activity
+            android:name="com.example.demo_spinner.MainActivity"
+            android:label="@string/app_name"
+            android:screenOrientation="portrait" >
+        </activity>
```

Comment:
```suggestion
```
The activity tag is already there, you should only have to add `android:screenOrientation="portrait"` to its attributes. Don't have to add `android:name` and `android:label`.
```diff
- android:name="com.example.demo_spinner.MainActivity"
- android:label="@string/app_name"
+ android:screenOrientation="portrait"
```